src/contexts/DarkModeContext/useLocalStorage.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+
+const useLocalStorage = (key, initialValue) => {
```

Comment:
fajny ten hook, dokładnie o to chodzi w custom hookach - bardziej złożone operacje, ale dotyczące jednej funkcjonalności